Interfaces/GEOS/geos_interface.F90

```diff
@@ -1276,7 +1276,7 @@ SUBROUTINE GEOS_Diagnostics( GC, IMPORT,  EXPORT, Clock, Phase, &
     REAL, POINTER                :: LFR(:,:)      => NULL()
     REAL, POINTER                :: CNV_FRC(:,:)  => NULL()
 
-    CHARACTER(LEN=ESMF_MAXSTR)   :: OrigUnit
+    INTEGER                      :: previous_units
```

Comment:
Changes in this file will cause conflicts when I bring in the updates for GEOS but that's fine. I fixed the unit conversions when running GEOS on discover but haven't brought the changes back into the standard model yet.